src/model.py

```diff
@@ -403,10 +413,26 @@ async def generate(self, request):
             )
 
             async for output in response_iterator:
-                if response_sender.is_cancelled():
+                is_cancelled = response_state["is_cancelled"]
```

Comment:
I lack some understanding. Could you please clarify for non-stream case, when `response_state["is_cancelled"]` is updated/checked if the state was changed from `False` to `True` ?

Reply:
resolved offline